pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/GodskrivingOmsorgspoengEtter1991.kt

```diff
@@ -0,0 +1,146 @@
+package no.nav.pensjon.brev.maler.redigerbar
+
+import no.nav.pensjon.brev.api.model.Sakstype
+import no.nav.pensjon.brev.api.model.TemplateDescription
+import no.nav.pensjon.brev.api.model.maler.Brevkode
+import no.nav.pensjon.brev.api.model.maler.legacy.redigerbar.OmsorgLegacyDataDto
+import no.nav.pensjon.brev.api.model.maler.legacy.redigerbar.OmsorgLegacyDataDtoSelectors.PesysDataSelectors.omsorgLegacyData
+import no.nav.pensjon.brev.api.model.maler.legacy.redigerbar.OmsorgLegacyDataDtoSelectors.pesysData
+import no.nav.pensjon.brev.maler.fraser.common.Constants.NAV_KONTAKTSENTER_TELEFON_PENSJON
+import no.nav.pensjon.brev.maler.fraser.common.Constants.NAV_URL
+import no.nav.pensjon.brev.maler.fraser.common.Felles
+import no.nav.pensjon.brev.maler.legacy.grunnlag_omsorggodskrgrunnlagliste_omsorggodskrgrunnlagar
+import no.nav.pensjon.brev.model.format
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.Language.English
+import no.nav.pensjon.brev.template.RedigerbarTemplate
+import no.nav.pensjon.brev.template.dsl.createTemplate
+import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
+import no.nav.pensjon.brev.template.dsl.languages
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brev.template.dsl.textExpr
+import no.nav.pensjon.brev.template.includePhrase
+import no.nav.pensjon.brevbaker.api.model.LetterMetadata
+
+@TemplateModelHelpers
+object GodskrivingOmsorgspoengEtter1991 : RedigerbarTemplate<OmsorgLegacyDataDto> {
+
+    // PE_IY_05_201
+    override val kode = Brevkode.Redigerbar.GODSKRIVING_AV_PENSJONSOPPTJENING_FOR_OMSORG_BARN_ETTER_1991
+    override val kategori = TemplateDescription.Brevkategori.OMSORGSOPPTJENING
+    override val brevkontekst = TemplateDescription.Brevkontekst.VEDTAK
+    override val sakstyper = setOf(Sakstype.OMSORG)
+
+    override val template = createTemplate(
+        name = kode.name,
+        letterDataType = OmsorgLegacyDataDto::class,
+        languages = languages(Bokmal, English),
+        letterMetadata = LetterMetadata(
+            displayTitle = "Godskriving av pensjonsopptjening (omsorgsopptjening) fordi du har omsorg for små barn",
+            isSensitiv = false,
+            distribusjonstype = LetterMetadata.Distribusjonstype.VIKTIG,
+            brevtype = LetterMetadata.Brevtype.INFORMASJONSBREV
+        )
+    ) {
+        //IF(   PE_pebrevkode = "PE_IY_05_201"    AND      (FF_GetArrayElement_Integer(PE_Grunnlag_OmsorgGodskrGrunnlagListe_OmsorgGodskrGrunnlagAr,1) > 1991      OR Year(PE_PersonSak_Fodselsdato) < 1948      OR Year(PE_PersonSak_Fodselsdato) > 1953)    )THEN      INCLUDE ENDIF
```

Comment:
Ser du har ett par kommentarer med exstream kode, men ingen showIf. Skal dette styres ved at noen bestiller riktig brev bare istedenfor? Da kan du evt slette logikk-komentarene om det stemmer.

Reply:
Ja, det er tanken - det var mykje logikk her for å skilje tekst til desse tre breva.

Men lurer på om det eigentleg burde vore logikk i PEN for å seie kva av breva som skal vera tilgjengeleg når, det er jo styrt ut frå data vi har (fødselsdato samt når omsorgs-året var) 🤔